### DISCUSSION TITLE
Display the dialogue title in the <title> tag

### DISCUSSION THREAD

**Discussion by CNSeniorious000** [anse-app/anse#25] [Ideas]
I think it is possible to display the conversation title in the <title> tag, similar to what is done on official websites. Here is an example of [what I have implemented myself](https://anse.up.railway.app/):

![image](https://github.com/anse-app/anse/assets/74518716/c17006fb-5460-4280-8e41-29dfc557d6bc)

However, due to the complexity of the file directory in this project, I have not determined the most appropriate location to add this logic.

On the other hand, modifying document.title in createEffect may not be the most appropriate way to change the title. Compared to SolidJS, I am more familiar with Svelte. In a Svelte application, I may create a svelte:head block in the Layout file which contains a <title> tag, and then use a store to set it globally.

However, upon reflection, isn't document.title itself similar to a store? Perhaps more encapsulation is not appropriate. What do you think about this?

<details><summary>appendix: screenshots of results of my implementation</summary>
<p></p>
<p><img src="https://github.com/anse-app/anse/assets/74518716/afe3e072-81ad-483e-a288-e57c57707317" alt="Landing page"></p>
<p><img src="https://github.com/anse-app/anse/assets/74518716/9203afd6-9430-4b44-ab1c-b172e5a275ac" alt="New Conversation"></p>
<p><img src="https://github.com/anse-app/anse/assets/74518716/b885f16a-bbc5-472a-b2d4-988ebdabc660" alt="After sending a message"></p>
</details>

**Comment by ddiu8081** [comment]
Nice try, I think using createEffect should be a good way to go for now. Feel free to create a PR if you like.
